canonical-web-and-design/ubuntu.com
# Update server download

## Done
Add the server weekly mailing list to the server thank you page

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure the copy matches [the copydoc](https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit)
- Check the design matches [the design](https://user-images.githubusercontent.com/36884067/69065492-c5f64c80-0a17-11ea-84c7-ca3a6c573173.jpg)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1748
